packages/sdk/src/protocols/lightpush/index.ts

```diff
@@ -89,16 +97,23 @@ class LightPushSDK extends BaseProtocolSDK implements ILightPushSDK {
           successes.push(success);
         }
         if (failure) {
+          failures.push(failure);
           if (failure.peerId) {
-            try {
-              await this.renewPeer(failure.peerId);
-              log.info("Renewed peer", failure.peerId.toString());
-            } catch (error) {
-              log.error("Failed to renew peer", error);
+            const peer = this.connectedPeers.find((connectedPeer) =>
+              connectedPeer.id.equals(failure.peerId)
+            );
+            if (peer) {
```

Comment:
I think here we should retry to any peer available - otherwise we loose message if peer got dropped (renewed, went offline or just networking issue)

Reply:
hm, I think we should align here on what is desired behavior.

considering what we care is successfully pushing a message at this stage (i.e no errors while sending)
then it is enough for us to have at least 1 successful push - in that case no retries needed
if all failures - then just retry but not necessarily to the same peer, just any peer
and if during all of this any peer is failing 3 times - we renew 

I probably summarized it for myself only, but just clarifying

Reply:
decoupled into follow up https://github.com/waku-org/js-waku/issues/2140